client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/problem/rpc/MockDatabaseDto.java

```diff
@@ -0,0 +1,54 @@
+package org.evomaster.client.java.controller.api.dto.problem.rpc;
+
+/**
+ * mock object for database
+ */
+public class MockDatabaseDto {
+
+    /**
+     * a key refers to the Web API
+     */
+    public String appKey;
+
+    /**
+     * it refers to a type of database, eg, mybatis
```

Comment:
??? mybatis is a library to deal with DB migrations... eg, like flyway.
why is it considered here as database type?